EIPS/eip-725.md

```diff
@@ -1,55 +1,93 @@
----
+```
 eip: 725
-title: Proxy Account
-author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
+title: ERC-725 Smart Contract Based Account
+author: Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
```

Comment:
If you are the primary author, I recommend leaving your GitHub username in here as the bot struggles with email addresses sometimes.  If you want both you can do:
```suggestion
author: Fabian Vogelsteller (@frozeman), Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
```

Reply:
good point